<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeBattleCircumstancesFormClass</class>
 <widget class="QDialog" name="ChangeBattleCircumstancesFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeBattleCircumstancesForm</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0" rowspan="6" colspan="3">
     <widget class="QListWidget" name="CircListWidget"/>
    </item>
    <item row="0" column="0" colspan="3" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Circumstances</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="6">
     <widget class="QListWidget" name="AppliedCircs"/>
    </item>
    <item row="0" column="5" rowspan="7">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="ToMenu">
        <property name="styleSheet">
         <string notr="true">background-color:#F0FFFF;
font-weight:800;
color:black;
</string>
        </property>
        <property name="text">
         <string>Go back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ApplyCircButton">
        <property name="text">
         <string>Apply circumstance</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ApplySpecialCircs">
        <property name="text">
         <string>Apply special</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RemoveGeneralCircButton">
        <property name="text">
         <string>remove general</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="3" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>General circumstances </string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Army 1 special circumstance:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="Army1Circ"/>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Army 2 special circumstance:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="Army2Circ"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
